doc/whatsnew/2.13.rst

```diff
@@ -279,6 +279,11 @@ Other Changes
 
   Closes #3781
 
+* Fix false positive - Allow unpacking of ``self`` in a subclass of ``typing.NamedTuple``.
+
+  Closes #5312
+  Closes #5671
```

Comment:
Hmm I wonder if we should add a changelog entry for the change introduce in this one as well. I'll leave these changelog fixes to @Pierre-Sassoulas's discretion. 

I'm approving the rest of it!